app/src/organisms/OnDeviceDisplay/RunningProtocol/__tests__/CancelingRunModal.test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import { renderWithProviders } from '@opentrons/components'
+import { i18n } from '../../../../i18n'
+import { CancelingRunModal } from '../CancelingRunModal'
+
+const render = () => {
+  return renderWithProviders(
+    <MemoryRouter>
+      <CancelingRunModal />
+    </MemoryRouter>,
+    {
+      i18nInstance: i18n,
+    }
+  )
+}
+
+describe('CancelingRunModal', () => {
+  it('should render text and buttons', () => {
```

Comment:
Probably icon is better than button since there is no button on this screen.

```suggestion
  it('should render text and icon', () => {
```